src/ring_theory/tensor_product.lean

```diff
@@ -717,3 +717,70 @@ end monoidal
 end tensor_product
 
 end algebra
+
+namespace tensor_product
+
+variables {R M N S: Type*} [comm_semiring R] [add_comm_monoid M] [add_comm_monoid N] [semiring S]
+variables [module R M] [module R N] [algebra R S]
+
+/--
+If `S` is an `R`-algebra, for a pair of morphisms `f : M →ₗ[R] S`, `g : N →ₗ[R] S`,
+We may obtain a map `M ⊗[R] N →ₗ[R] S` via `a ⊗ b ↦ f(a) * g(b)`.
+This may not be that useful in module-sense, but is useful when `M`, `N` are both algebras.
```

Comment:
I'm not sure what "in module-sense" means.

Reply:
I meant that it is not that useful when `M` `N` are merely modules.
But now the part is completely removed.